massdriver-package-alarm/main.tf

```diff
@@ -0,0 +1,10 @@
+resource "massdriver_package_alarm" "main" {
+  cloud_resource_id = var.cloud_resource_id
+  display_name      = var.display_name
+  metric {
+    name       = var.metric_name
+    namespace  = var.metric_namespace
+    statistic  = var.statistic
```

Comment:
I could go either way but my thoughts here are.

1. We can be consistent. 
  + `metric_name`, `metric_namespace` , `metric_statistic`, `metric_dimensions` 

Or a variable?

```
variable "metric" {
  type = object({
    name = string 
    namespace = string
    statistic = string
    dimensions = any
  })
}
```

Reply:
I tried this variable and ran into issues. I prefer using `metric_statistic` and `metric_dimensions`